### ISSUE TITLE
Operations [Renumeration]: Team Payments for JUNE.

### ISSUE THREAD

**Issue by wolstaeb** [C3ETH/c3eth-operations#230]
As per our consensus and #102, pay @C3ETH/c3eth-team remuneration for June 2022. Perform the payment in one transaction for transparency and accountability purposes.

- [x] #102. Please note the full budget for Q3 is still under draft and needs to be fully approved. June payments are done it advance of this being finalised. Budget is set at $15OUSD a week.
- [x] Get the 7-Day Average USD/ADA price from [coinmarketcap.com](https://bit.ly/3fVrYKT). See details below.
- [x] [Update compensation spreadsheet](https://bit.ly/3H1G8G7) with exchange rate, and
- [x] Work out Ada/Lovelace amounts
- [x] Double Check amounts and put them into payment script
- [x] [Transfer total amount needed](https://explorer.cardano.org/en/transaction?id=7f9fdc3edde699e2b889ed9d0c0e590c09b11aee3b37d998ccf5f612213b90e0) to the local payment address.
- [x] Update script with input transaction details
- [x] [Perform the renumeration payment](https://explorer.cardano.org/en/transaction?id=af7b782ba01fabc697dae880f891ecb9668c1f75ff2f6f077b497c33cc9c360d)
- [x] [Notify the @C3ETH/c3eth-team of the transaction](https://discord.com/channels/868850218995707914/875343499824889947/988310096599728208).
- [x] Backup payment script details.
- [x] Document transaction details

---
## Transaction Details

Links above document the payment plan and the two associated transactions.

### Coin marketcap 7-Day rate

For Payment we use a simple seven day average to devise the payment conversion rates at the approximate time (within 30mins of) the payments being made. The USD/ADA rate is calculated using the following data points from Coinmarket Cap 7D prices. The intention is to smooth out fluations in the USD/ADA exchange rates.

```
ADA-7DA = (7DOPEN + 7DCLOSE + HIGH + LOW) / 4
```

Where: 7DOPEN, 7DCLOSE, HIGH, LOW are USD/ADA prices. HIGH and LOW are the Coinmarket Cap 7d Low/High, and the 7DSTART and 7DCLOSE are taken from the seven day window of the chart below.

PERIOD | 7DOPEN | 7DCLOSE | 7DLOW | 7DHIGH | AVG
-- | -- | -- | -- | -- | --
June | 0.47 | 0.4661 | 0.4285 | 0.5414 | 0.4765

![Ada 7DAY 2022-06-20 02 50 38 PM](https://user-images.githubusercontent.com/82072426/174516721-5d644c10-b2f5-45db-b955-9716ee71b722.png)

**Comment by wolstaeb** [comment]
**NOTE**: There was an error in the payment amounts due to an error in the spreadsheet formula. Team members have been underpaid $423USD instead of the $480USD due. The extra adjustment of $57USD has been added to June team payments. 

Added to the base was the adjustment for $57USD which was missing from April payments... Unfortunately I made a error, adding the $57USD to each week, rather than just a one off payment. Everyone has been paid **$57USDx3 ($171USD)** too much. Rather than requesting the funds back, future payments will adjust for this over payment - without error this time. ;-)
